sdks/v4-sdk/src/PositionManager.ts

```diff
@@ -257,6 +262,66 @@ export abstract class V4PositionManager {
     }
   }
 
+  /**
+   * Produces the calldata for completely or partially exiting a position
+   * @param position The position to exit
+   * @param options Additional information necessary for generating the calldata
+   * @returns The call parameters
+   */
+  public static removeCallParameters(position: Position, options: RemoveLiquidityOptions): MethodParameters {
+    /**
+     * cases:
+     * - if liquidityPercentage is 100%, encode BURN_POSITION and then TAKE_PAIR
```

Comment:
this doesnt actually seem to be the logic its following?
Its not doing `if options.liquidityPercentage.equalTo(ONE) -> burn, else -> decrease`